src/App.tsx

```diff
@@ -1,185 +1,343 @@
-    />;
-    case 'feedback2': return <FeedbackScreen2
-      response={getLastResponse()}
-      indicatorsLastTurn={getIndicatorsLastMonth()}
-      onClickContinue={() => ending ? show('end') : show('event')}
-      onClickExtra={() => show('feedbackExtra')}
-      onClickSource={showSource}
-    />;
-    case 'feedbackExtra': return <FeedbackExtra
-      response={getLastResponse()}
-      onClickBack={() => show('feedback2')}
-      onClickSource={showSource}
-    />;
+    case "feedback1":
+      return (
+        <FeedbackScreen1
+          response={getLastResponse()}
+          feedback={getSocialFeedback(getLastResponse().socialMediaResponse)}
+          onClick={() => show("feedback2")}
+        />
+      );
+    case "feedback2":
+      return (
+        <FeedbackScreen2
+          response={getLastResponse()}
+          indicatorsLastTurn={getIndicatorsLastMonth()}
+          onClickContinue={() => (ending ? show("end") : show("event"))}
+          onClickExtra={() => show("feedbackExtra")}
+          onClickSource={showSource}
+        />
+      );
+    case "feedbackExtra":
+      return (
+        <FeedbackExtra
+          response={getLastResponse()}
+          onClickBack={() => show("feedback2")}
+          onClickSource={showSource}
+        />
+      );
 
     // End show
-    case 'end': return <Ending
-      leaderStyle={endings[ending]}
-      onClick={() => { show('allEndings') }}
-    />;
-    case 'allEndings': return <AllEndings
-      onClick={(name: string) => { setEnding(name); show('viewEnding') }}
-      onReplay={reset}
-    />;
-    case 'viewEnding': return <ViewEnding
-      leaderStyle={endings[ending]}
-      onClick={() => { show('allEndings') }}
-    />;
-    default: return <></>
+    case "end":
+      return (
+        <Ending
+          leaderStyle={endings[ending]}
+          onClick={() => {
+            show("allEndings");
+          }}
+        />
+      );
+    case "allEndings":
+      return (
+        <AllEndings
+          onClick={(name: string) => {
+            setEnding(name);
+            show("viewEnding");
+          }}
+          onReplay={reset}
+        />
+      );
+    case "viewEnding":
+      return (
+        <ViewEnding
+          leaderStyle={endings[ending]}
+          onClick={() => {
+            show("allEndings");
+          }}
+        />
+      );
+    default:
+      return <></>;
   }
-}
+};
+
+/**
+ * Inline styles are used on purpose so we don't need to load tailwind css.
+ */
+const Loading: React.FC = () => {
+  return (
+    <div
+      style={{
+        flexDirection: "row",
+        justifyContent: "center",
+        alignItems: "center",
+        display: "flex",
+        height: "95vh",
+      }}
+    >
+      <div className="lds-ring">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+    </div>
+  );
+};
+
+const preloadAssets = () => {
+  const assets = [
+    new FontFaceObserver("Bebas Neue").load(),
```

Comment:
The assets we wish to preload must be declared here, so if any new images are added or changed, this will need to be updated.